src/routes/statusPage/index.js

```diff
@@ -0,0 +1,187 @@
+import { h } from 'preact';
+import { Button, Text, Box, Flex, Grid, GridItem, Input } from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import { useState, useEffect } from 'preact/hooks';
+import axios from 'axios';
+import { Link} from 'preact-router';
+import BASE_URL from '../../config/api/constant.js';
+import { getUserToken } from '../../config/api/auth.js';
+
+const StatusPage = () => {
+    const [errorMessageSaveUrl, setErrorMessageSaveUrl] = useState('');
+    const [categoryList, setCategoryList] = useState([]);
+    const [path, setPath] = useState('');
+
+    const {
+        handleSubmit,
+        register,
+        setValue,
+    } = useForm({});
+
+    const {
+        register: register2,
+        setValue: setValue2,
+        handleSubmit: handleSubmit2,
+      } = useForm({});
+
+    useEffect(()=> {
+		axios.get(`${BASE_URL}/status-page/config/`, {
+			headers: {
+				Authorization:`Token ${getUserToken()}`
+			}
+		}).then((response)=>{
+			setValue("path", response.data["path"])
+            setPath(response.data['path'])
+		})
+    }, [])
+
+    useEffect(()=> {
+		axios.get(`${BASE_URL}/status-page/category/`, {
+			headers: {
+				Authorization:`Token ${getUserToken()}`
+			}
+		}).then((response)=>{
+			setCategoryList(response.data)
+		})
+    }, [categoryList])
```

Comment:
```suggestion
    }, [])
```

potential inifinite loop (?)

Reply:
done